sunpy/map/tests/conftest.py

```diff
@@ -141,14 +139,19 @@ def eit_test_map():
     """
     return get_dummy_map_from_header(get_test_filepath("EIT_header/efz20040301.020010_s.header"))
 
+
 @pytest.fixture
 def sample_171():
     from sunpy.data.sample import AIA_171_IMAGE
 
     return sunpy.map.Map(AIA_171_IMAGE)
 
+
 @pytest.fixture
 def sample_hmi():
     from sunpy.data.sample import HMI_LOS_IMAGE
 
     return sunpy.map.Map(HMI_LOS_IMAGE)
+
+
+simple_map = pytest.fixture(make_simple_map)
```

Comment:
I'd rather have this under the function, it's pretty weird that it's separate anyway. Keeping it there will make it more obvious.

Reply:
Fixed.